airbyte-integrations/connectors/destination-databend/destination_databend/writer.py

```diff
@@ -0,0 +1,127 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from collections import defaultdict
+from datetime import datetime
+from itertools import chain
+
+from airbyte_cdk import AirbyteLogger
+from destination_databend.client import DatabendClient
+
+
+class DatabendWriter:
+    """
+    Base class for shared writer logic.
+    """
+
+    flush_interval = 1000
+
+    def __init__(self, client: DatabendClient) -> None:
+        """
+        :param client: Databend SDK connection class with established connection
+            to the databse.
+        """
+        self.client = client
+        self.cursor = client.open()
+        self._buffer = defaultdict(list)
+        self._values = 0
+
+    def delete_table(self, name: str) -> None:
+        """
+        Delete the resulting table.
+        Primarily used in Overwrite strategy to clean up previous data.
+
+        :param name: table name to delete.
+        """
+        cursor = self.cursor
```

Comment:
why is this assignment needed cant you invoke `self.cursor.execute(f"DROP TABLE IF EXISTS _airbyte_raw_{name}")` directly?

Reply:
Thanks a lot. 😄